Classes/Event/EventPublisher.php

```diff
@@ -106,6 +107,9 @@ public function publishMany(string $streamName, array $events, int $expectedVers
             $eventClassName = $this->eventTypeResolver->getEventClassNameByType($rawEvent->getType());
             $event = $this->propertyMapper->convert($rawEvent->getPayload(), $eventClassName, $configuration);
             foreach ($this->eventListenerLocator->getSynchronousListeners($rawEvent->getType()) as $listener) {
+                if ($listener[0] instanceof ActsBeforeInvokingEventListenerMethodsInterface) {
```

Comment:
Just in case you missed it before: `$listener` is not necessarily an array in the form `[<object>, '<methodName>']` but could be any `\callable`. (just mentioning it for the record, I'm not sure how we could solve it)